app/controllers/katello/concerns/api/v2/repository_content_controller.rb

```diff
@@ -150,7 +150,7 @@ def check_repo_for_content_resource
 
       def filter_by_content_view_filter(filter, options)
         ids = filter.send("#{singular_resource_name}_rules").map(&:uuid)
-        repo_ids = filter.applicable_repos.readable.select([:pulp_id, "#{Katello::Repository.table_name}.name"])
+        repo_ids = filter.applicable_repos.readable.map(&:pulp_id)
```

Comment:
you might use  .pluck("#{Repository.table_name}.pulp_id")  as that should be faster